### DISCUSSION TITLE
Expected release date for a new version

### DISCUSSION THREAD

**Discussion by lsarni** [react-native-elements/react-native-elements#3910] [Q&A] [answered]
The current version was released 10 months ago. A lot of bug fixes have been merged since then, and we have been patching the package in our project when we bump into them. 

Is there a date for when a new version will be released?

**Comment by GaylordP** [comment]
An excellent question ^^

**Comment by ChromeQ** [comment]
Seems like this is dead, what do others recommend as an alternative?

**Comment by theianmay** [comment, marked as answer]
## 🚀 React Native Elements v5 Beta is Live!  

Hello everyone! 👋 Thank you all for your patience while our new team has been hard at work. 
We’re excited to announce that **v5.0.0-beta.1** is now available! 

This is a **beta release**, intended for testing and feedback. The **full stable v5.0.0** is still in the pipeline.  

---
###⚠️ What you need to know
- This is not yet stable, intended for testing and feedback.
- For example, a known issue is: Button disabled state styling on Android. Workarounds and discussion available here: #3985
- Associated issue: https://github.com/facebook/react-native/issues/53849
- You can install the beta via NPM:
```bash
npm install @rneui/base@beta @rneui/themed@beta
```
---

### 📝 Feedback
Please provide **bugs, suggestions, or feedback**:  
- GitHub Discussion: https://github.com/react-native-elements/react-native-elements/discussions/4007 
- Discord: https://discord.com/invite/e9RBHjkKHa

**Next Steps:**  
- I will be closing this for now to keep things organized.
- All ongoing beta feedback should go to the Discussion or Discord.
